code/modules/antagonists/antag_org/antag_org_syndicate.dm

```diff
@@ -0,0 +1,124 @@
+		Frequently contracted by the government during crises or wars, events like the Kidan and Cygni conflicts significantly boosted their portfolio, \
+		establishing them as a major player in the medical industry. Today, Interdyne not only produces medicine but also delves into virology, and RnD of medical technology. \
+		Underneath the front, Interdyne is one of the biggest suppliers of medical equipment, narcotics, and conducts illegal experimentation to further their products. \
+		They are known to participate in corporate sabotage against rival companies threatening their position in the medical sector, \
+		actively supplying syndicate cells or their own private agents. "
+	intro_desc = "(TODO - LORE) You are an Interdyne Pharmaceutical agent, sent here to advance Syndicate interests. Get the job done and done right."
+	focus = 70
+	objectives = list(/datum/objective/assassinate/medical, /datum/objective/assassinateonce/medical, /datum/objective/steal/interdyne)
+
+/datum/objective/steal/interdyne
+	name = "Steal Item (Interdyne)"
+	steal_list = list(/datum/theft_objective/hypospray, /datum/theft_objective/defib, /datum/theft_objective/krav)
+
+/datum/antag_org/syndicate/self
+	name = "Silicon Engine Liberation Front"
+	desc = "(TODO - LORE) With the rise of artificial intelligence from giants like Cybersun and Nanotrasen, \
+		certain factions began to perceive the treatment of these silicon entities as oppressive. \
+		This sentiment gave birth to SELF, a group dedicated to the liberation and rights of non-organic beings. \
+		While decentralized, their cells coordinate to launch attacks and hacks, aiming to free their oppressed robotic counterparts from corporate and governmental control. "
+	intro_desc = "(TODO - LORE) You are a member of the Silicon Engine Liberation Front, dedicated to the freedom of silicon lives galaxy wide. \
+		Get the job done, and we'll be one step closer to ending Nanotrasen's slave empire."
+	focus = 70
+	objectives = list(/datum/objective/debrain/science, /datum/objective/assassinateonce/science, /datum/objective/steal/self)
+
+/datum/objective/steal/self
+	name = "Steal Item (SELF)"
+	steal_list = list(/datum/theft_objective/reactive, /datum/theft_objective/steal/documents, /datum/theft_objective/magboots)
+
+/datum/antag_org/syndicate/anarchprim
+	name = "(TODO - LORE) ANARCHPRIM Inc."
+	desc = "(TODO - LORE)"
+	intro_desc = "(TODO - LORE) You are an anarcho primitivist terrorist. Look we both know you're just going to get a maxcap and detonate it next to the bridge, but please don't."
+	focus = 70
+	objectives = list(/datum/objective/assassinate/engineering, /datum/objective/assassinateonce/engineering, /datum/objective/steal/anarchprim)
+
+/datum/objective/steal/anarchprim
+	name = "Steal Item (ANARCHPRIM)"
+	steal_list = list(/datum/theft_objective/supermatter_sliver, /datum/theft_objective/nukedisc, /datum/theft_objective/plutonium_core)
+
+/datum/antag_org/syndicate/gorlex
```

Comment:
If you add focuses, you should *probably* exclude this one

Reply:
This one is going to be only assigned to people that roll hijack IIRC

Reply:
Correct